src/main/java/org/jenkinsci/plugins/workflow/multibranch/ResolveScmStep.java

```diff
@@ -25,25 +25,21 @@
 
 package org.jenkinsci.plugins.workflow.multibranch;
 
-import edu.umd.cs.findbugs.annotations.CheckForNull;
-import edu.umd.cs.findbugs.annotations.NonNull;
-import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
```

Comment:
We will actually want to revert this to match https://github.com/jenkinsci/jenkins/pull/4604, but it can wait.

Reply:
FTR I originally migrated these to fix a deprecation warning. Apparently these classes we un-deprecated in spotbugs/spotbugs#180. Not sure if that commit has propagated to recent Jenkins plugin builds yet or not.

Reply:
Yes, older versions of the lib were deprecated. Newer versions are not. It is complicated.

Reply:
@jtnord please advise w.r.t. `plugin-pom`

Reply:
the plugin pom should be fine, the issue is the Jenkins baseline.
all versions up to 2.236 use spotbugs 3.x 2.237 onwards use 4.x

Reply:
So to confirm we are stuck with the wrong annotation dependency until `jenkins.version` is 2.237+, regardless of `plugin-pom` version?

Reply:
> we are stuck with the wrong annotation dependency

you can override the dependency version in your own pom.